<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyUI</class>
 <widget class="QMainWindow" name="MyUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MyUI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="MyUiMainStack">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>600</width>
      <height>500</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QStackedWidget
{
	background-color: rgb(113, 113, 113);
}</string>
    </property>
    <widget class="QWidget" name="PgHome">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>160</y>
        <width>58</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>home</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pgExplore">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>150</y>
        <width>58</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>explore</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="PgDevices">
     <widget class="QFrame" name="FrameDevPg">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>580</width>
        <height>480</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="PgSettings">
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>220</y>
        <width>58</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>settings</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pbFunction_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: rgb(255, 255, 255);
	color: rgb(0, 0, 0);
	font: 10pt &quot;Ubuntu&quot;;
	border-radius: 0px;
}

QPushButton:Pressed
{
	background-color: rgb(0, 0, 0);
	color: rgb(255, 255, 255);
	font: 10pt &quot;Ubuntu&quot;;
	text-decoration:underline;
}</string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbFunction_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: rgb(255, 255, 255);
	color: rgb(0, 0, 0);
	font: 10pt &quot;Ubuntu&quot;;
	border-radius: 0px;
}

QPushButton:Pressed
{
	background-color: rgb(0, 0, 0);
	color: rgb(255, 255, 255);
	font: 10pt &quot;Ubuntu&quot;;
	text-decoration:underline;
}</string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbFunction_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: rgb(255, 255, 255);
	color: rgb(0, 0, 0);
	font: 10pt &quot;Ubuntu&quot;;
	border-radius: 0px;
}

QPushButton:Pressed
{
	background-color: rgb(0, 0, 0);
	color: rgb(255, 255, 255);
	font: 10pt &quot;Ubuntu&quot;;
	text-decoration:underline;
}</string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbFunction_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: rgb(255, 255, 255);
	color: rgb(0, 0, 0);
	font: 10pt &quot;Ubuntu&quot;;
	border-radius: 0px;
}

QPushButton:Pressed
{
	background-color: rgb(0, 0, 0);
	color: rgb(255, 255, 255);
	font: 10pt &quot;Ubuntu&quot;;
	text-decoration:underline;
}</string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbFunction_5">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>110</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: rgb(255, 255, 255);
	color: rgb(0, 0, 0);
	font: 10pt &quot;Ubuntu&quot;;
	border-radius: 0px;
}

QPushButton:Pressed
{
	background-color: rgb(0, 0, 0);
	color: rgb(255, 255, 255);
	font: 10pt &quot;Ubuntu&quot;;
	text-decoration:underline;
}</string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbFunction_6">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>240</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: rgb(255, 255, 255);
	color: rgb(0, 0, 0);
	font: 10pt &quot;Ubuntu&quot;;
	border-radius: 0px;
}

QPushButton:Pressed
{
	background-color: rgb(0, 0, 0);
	color: rgb(255, 255, 255);
	font: 10pt &quot;Ubuntu&quot;;
	text-decoration:underline;
}</string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbFunction_7">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>360</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: rgb(255, 255, 255);
	color: rgb(0, 0, 0);
	font: 10pt &quot;Ubuntu&quot;;
	border-radius: 0px;
}

QPushButton:Pressed
{
	background-color: rgb(0, 0, 0);
	color: rgb(255, 255, 255);
	font: 10pt &quot;Ubuntu&quot;;
	text-decoration:underline;
}</string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbFunction_8">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>470</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: rgb(255, 255, 255);
	color: rgb(0, 0, 0);
	font: 10pt &quot;Ubuntu&quot;;
	border-radius: 0px;
}

QPushButton:Pressed
{
	background-color: rgb(0, 0, 0);
	color: rgb(255, 255, 255);
	font: 10pt &quot;Ubuntu&quot;;
	text-decoration:underline;
}</string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbFunction_9">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>565</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: rgb(255, 255, 255);
	color: rgb(0, 0, 0);
	font: 10pt &quot;Ubuntu&quot;;
	border-radius: 0px;
}

QPushButton:Pressed
{
	background-color: rgb(0, 0, 0);
	color: rgb(255, 255, 255);
	font: 10pt &quot;Ubuntu&quot;;
	text-decoration:underline;
}</string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbFunction_10">
    <property name="geometry">
     <rect>
      <x>285</x>
      <y>565</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: rgb(255, 255, 255);
	color: rgb(0, 0, 0);
	font: 10pt &quot;Ubuntu&quot;;
	border-radius: 0px;
}

QPushButton:Pressed
{
	background-color: rgb(0, 0, 0);
	color: rgb(255, 255, 255);
	font: 10pt &quot;Ubuntu&quot;;
	text-decoration:underline;
}</string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbFunction_11">
    <property name="geometry">
     <rect>
      <x>435</x>
      <y>565</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: rgb(255, 255, 255);
	color: rgb(0, 0, 0);
	font: 10pt &quot;Ubuntu&quot;;
	border-radius: 0px;
}

QPushButton:Pressed
{
	background-color: rgb(0, 0, 0);
	color: rgb(255, 255, 255);
	font: 10pt &quot;Ubuntu&quot;;
	text-decoration:underline;
}</string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbFunction_12">
    <property name="geometry">
     <rect>
      <x>584</x>
      <y>565</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: rgb(255, 255, 255);
	color: rgb(0, 0, 0);
	font: 10pt &quot;Ubuntu&quot;;
	border-radius: 0px;
}

QPushButton:Pressed
{
	background-color: rgb(0, 0, 0);
	color: rgb(255, 255, 255);
	font: 10pt &quot;Ubuntu&quot;;
	text-decoration:underline;
}</string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QLabel" name="bottomKeyInd_1">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>550</y>
      <width>150</width>
      <height>3</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	background-color: rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="bottomKeyInd_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>550</y>
      <width>150</width>
      <height>3</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	background-color: rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="bottomKeyInd_3">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>550</y>
      <width>150</width>
      <height>3</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	background-color: rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="bottomKeyInd_4">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>550</y>
      <width>150</width>
      <height>3</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	background-color: rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
